src/Components/Onboarding/Steps/Genes/SuggestedGenes.tsx

```diff
@@ -115,17 +115,15 @@ class SuggestedGenesContent extends React.Component<Props, null> {
   }
 
   render() {
-    const items = this.props.suggested_genes.map((item, index) => (
-      <LinkContainer>
+    const items = this.props.suggested_genes.map(item => (
+      <LinkContainer key={item._id}>
```

Comment:
This is the only component that needs a `key`, as it’s the one that has siblings, but funnily enough fixing this uncovered that we actually have a duplicate gene in [the list of suggested genes](https://metaphysics-production.artsy.net/?query=%7B%0A%20%20suggested_genes%20%7B%0A%20%20%20%20id%0A%20%20%7D%0A%7D&variables=%7B%7D), namely `textile-arts`.

cc @xtina-starr @yuki24

Reply:
The [`react-transition-group`](https://github.com/reactjs/react-transition-group), which is used by the `<ReplaceTransition>` component, relies on the `key` prop both on [v2](https://github.com/reactjs/react-transition-group/blob/1437e6731e4bc15850d987bb063e24020ec24061/src/TransitionGroup.js#L129-L130) and [v1](https://github.com/reactjs/react-transition-group/blob/v1.2.1/src/TransitionGroup.js#L41-L42). I agree the `<LinkContainer>` should have a key prop and It's probably ok to take out the `key` from `<ReplaceTransition>`, but would the animation work without `key` in the `<ItemLink>`?

Reply:
That's a good point @yuki24 I'll merge this PR into mine and see if it still works as expected.

Reply:
Thanks for noticing that @yuki24!

If it does turn out to be broken with this change, then we should have a test that mentions why those are needed and maybe also a comment here?